Providers/Scripts/2.4x-2.5x/Scripts/Tests/dummy_nxOMSAutomationWorker_files/oms_conf_auto.conf

```diff
@@ -0,0 +1,9 @@
+[oms-worker-conf]
+auto_registered_worker_conf_path = /home/bhbrahma/PowerShell-DSC-for-Linux/Providers/Scripts/2.6x-2.7x/Scripts/Tests/temp/worker.conf
+
+[oms-global]
+resource_version = 1.4
```

Comment:
Please make sure the version used in all files is consistent (your Makefile says 1.5 at the moment); otherwise, please explain the inconsistencies.